### ISSUE TITLE
Question about the results on ScanNet

### ISSUE THREAD

**Issue by SamMaoYS** [facebookresearch/scenescript#9]
Hi @chrisdxie,

Thank you for sharing SceneScript—it’s a great project and very inspiring!

I have a question regarding the ScanNet results table. Are these results from a zero-shot evaluation on ScanNet using a model trained on the ASE dataset, or was the model trained/finetuned on ScanNet?

Looking forward to your insights.

All the best,
Yongsen Mao

**Comment by chrisdxie** [comment]
Hi @SamMaoYS, thanks for the question. The model was pre-trained on ASE, then finetuned on ScanNet. Let me know if you have more questions.

**Comment by SamMaoYS** [comment]
Hi @chrisdxie,
Thank you so much for the clarifications!

I have one additional question regarding the ScanNet results: Since the ground truth for the original ScanNet test set is hidden, I assume that the reported F1 score in the table corresponds to the original validation set. Is that correct?

Looking forward to your response. Thanks again!

**Comment by chrisdxie** [comment]
Yes, that's correct. Thanks for clarifying!
